File: adt_easy_20250422_1/q_A.kt
import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)
    var line= sc.nextLine()

    if (line.contains("ACE")
        || line.contains("BDF")
        || line.contains("CEG")
        || line.contains("DFA")
        || line.contains("EGB")
        || line.contains("FAC")
        || line.contains("GBD"))
        println("Yes")
    else
        println("No")
}

File: adt_easy_20250422_1/q_A.rs
use std::io::{stdin, BufRead};

fn main() {
    let std= stdin();
    let mut buf= std.lock().lines();
    let s= buf.next().unwrap().unwrap();

    match s.as_str() {
        "ACE" | "BDF" | "CEG" | "DFA" | "EGB" | "FAC" | "GBD" => println!("Yes"),
        _ => println!("No")
    }
}
